### PR TITLE
Feat/overview

### PR DESCRIPTION
**Pull request by me-jterhorst**
Create a single page overview to access the contents of tasks. 
Implement single Page into Routing
Build contents of single page from mock data

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import PasswordReset from "./pages/PasswordReset";
 import CreateCase from "./pages/CreateCase";
 import CreateAction from "./pages/CreateAction";
 import SinglePage from "./pages/SinglePage";
+import Overview from "./pages/Overview";
 /* =========================== Import Requirements */
```
Comment (hugh-burgess) [inline client/src/App.js:15]: this could just be a blank space no?

File: client/src/App.js [hunk 2]
```diff
@@ -27,69 +28,71 @@ export default function App() {
...
             verb: "refuel",
             action: "the Batmobil",
             with: "Alfred",
           },
           {
             taskId: 3,
-            happening: "is starting a gang war",
             verb: "Call",
             action: "Robin",
             with: "Robin",
```
Comment (hugh-burgess) [inline client/src/App.js:98]: would this not suit better in a separate file like a JSON file? Rather than in `App.js`

**Review by hugh-burgess — COMMENTED** [review]
Looking good bud, couple things..

**Review by adrianammt — COMMENTED** [review]
It looks really good! Everything you mentioned it should work is working. I just notice one little thing. When I click on the menu icon and it slides in, I can't get out of it unless I click on an option. ;)

File: client/src/App.js (hunk 1 above)
Comment (me-jterhorst) [reply client/src/App.js:15]: There would always be the searchbar, but I catch your drift. 

I am implementing a notification if the fetched data returns an empty array or is still loading.

File: client/src/App.js (hunk 2 above)
Comment (me-jterhorst) [reply client/src/App.js:98]: Yes I am doing the frontend first before I build the API and Database

**Pull request merged**